src/core.js

```diff
@@ -114,10 +114,21 @@ function date(format) {
       timestamp = d.getTime();
     }
     var result = '';
+    var grab_next = false;
     for (var i = 0; i < format.length; i++) {
       var character = format.charAt(i);
+      if (grab_next) {
+          result += character;
+          grab_next = false;
+          continue;
+      }
       switch (character) {
 
+        case '\\':
+          grab_next = true;
+          console.log(grab_next);
+          continue;
+
```

Comment:
Well, `console.log(grab_next)` isn't supposed to be there actually.
When someone calls something like `date('Day: d')` the "D" and "a" are interpreted as tokens.  The backslash is supported in PHP as an escape token so that you can call `date('\D\ay: d)`.  The backslashes themselves need to be escaped though to work, so the call becomes `date('\\D\\ay: d)`.
The way this patch works is when it detects a backslash it grabs the next character literally rather than checking it for a token.